### PR TITLE
DDPB-1704 review report heading fix

### PR DESCRIPTION
**Pull request by Biggsen**
looking for '104' in the string, instead of an exact '104' to match 104-6.

### CODE REVIEW

File: src/AppBundle/Resources/views/Report/Formatted/formatted_body.html.twig [hunk 1]
```diff
@@ -17,6 +17,9 @@
     {% include 'AppBundle:Css:formatted-report.css.twig' %}
 </style>
 
+<p>{{ report.get104TransSuffix }}</p>
```
Comment (slizzio) [inline src/AppBundle/Resources/views/Report/Formatted/formatted_body.html.twig]: Is this meant to be here?

File: src/AppBundle/Resources/views/Report/Formatted/formatted_body.html.twig [hunk 2]
```diff
@@ -17,6 +17,9 @@
...
+<p>{{ report.type }}</p>
```
Comment (slizzio) [inline src/AppBundle/Resources/views/Report/Formatted/formatted_body.html.twig]: Are these meant to be here? Debug only?

Comment (Biggsen) [reply src/AppBundle/Resources/views/Report/Formatted/formatted_body.html.twig]: Debug only. Gone.

**Comment by Biggsen** [comment]
Not showing new commits.

**Pull request closed without merging**